### DISCUSSION TITLE
DS18B20 is redundant when using MPU6050

### DISCUSSION THREAD

**Discussion by covert-** [universam1/iSpindel#590] [Ideas]
Is the DS18B20 needed for temperature when the MPU6050 has silicon die temperature accurate to 1C ?

**Comment by ErikdBr** [comment]
The firmware of the iSpindel needs a DS18B20 in order to function. to my knowledge the temp sensor on the MPU6050 is only used to measure the temp of the board(die temperature) itself, not for it's environment.

"Die temperature sensor (DTS) is an electrical circuit in a silicon chip/ Die to measure its temperature while testing the device at various temperatures for its functionality before being shipped to the customer."

**Comment by universam1** [comment]
The temp sensor of the MPU is for internal use only aka DSP and it's off to the environment by several degrees and completely useless to track. 
This has been tested and discussed several times, should go into FAQ
